src/kitovu/utils.py

```diff
@@ -29,6 +29,19 @@ def get_password(plugin: str, identifier: str, prompt: str) -> str:
     return password
 
 
+def delete_password(plugin: str, identifier: str) -> None:
+    """Get the password for the given URL via keyring.
```

Comment:
Probably need to update this comment too :)

Reply:
Whups, thanks for the hint. I've changed that.